syntax_checkers/scala/ensime.vim

```diff
@@ -0,0 +1,35 @@
+let s:save_cpo = &cpo
```

Comment:
As I were afraid, when I moved `syntax_checkers` folder into sub-subfolder Syntastic failed to find the checker plugin, rendering this integration unusable.